pkg/sqlparser/parser.go

```diff
@@ -308,3 +308,38 @@ func AssetTypeToDialect(assetType pipeline.AssetType) (string, error) {
 	}
 	return dialect, nil
 }
+
+func (s *SQLParser) AddLimit(sql string, limit int) (string, error) {
```

Comment:
let's add tests for this as well to ensure we are calling the right function in the right way from the Go side.